src/molecules/link-card/link-card.test.tsx

```diff
@@ -1,3 +1,59 @@
+import React from "react";
+import faker from "faker";
+import { LinkCard } from "./link-card";
+import { TestUtils } from "../../tests/test-utils";
+import { LinkCardTypes } from "../constants/link-card-types";
+
 describe("LinkCard", () => {
-    test.skip("TODO - https://github.com/AndcultureCode/AndcultureCode.JavaScript.React.Components/issues/36", () => {});
+    test("When default props, renders LinkCard", () => {
+        // Arrange
+        const expected = faker.random.words();
+        const label = faker.random.words();
+
+        // Act
+        const { getByText } = TestUtils.renderWithRouter(
+            <LinkCard label={label}>{expected}</LinkCard>
+        );
+
+        //Assert
+        expect(getByText(expected)).not.toBeNull();
+    });
+
+    test("When type prop is button, renders LinkCard with button", () => {
+        // Arrange
+        const expected = faker.random.words();
+        const label = faker.random.words();
+        const type = LinkCardTypes.Button;
+
+        // Act
+        const { container } = TestUtils.renderWithRouter(
+            <LinkCard label={label} type={type}>
+                {expected}
+            </LinkCard>
+        );
+        const result = container.getElementsByTagName(LinkCardTypes.Button)[0];
+
+        // Assert
+        expect(result).not.toBeNull();
+        expect(result.classList).toContain("c-button");
+        expect(container.innerHTML).toContain("button");
```

Comment:
What is this testing, exactly?

Reply:
This is testing that the LinkCard is rendered with a button and not an a tag

Reply:
I'm not too familiar with `react-testing-library`, but if we're retrieving the DOM element with `getElementsByTagName(LinkCardTypes.Button)` (where `LinkCardTypes.Button` is `Button`), and we know it's not `null` or `undefined`, shouldn't that already be covered?

Reply:
Thanks for jumping on the call with me to discuss this. 👍